src/vs/gitpod/node/gitpodInsightsAppender.ts

```diff
@@ -214,6 +214,33 @@ function mapTelemetryData(eventName: string, data: any): RemoteTrackMessage | un
 					timestamp: data.timestamp
 				},
 			};
+		case 'notification:show':
+		case 'notification:close':
+		case 'notification:hide':
+			return {
+				event: `vscode_${formatEventName(eventName)}`,
+				properties: {
+					id: data.id,
+					source: data.source,
+					workspaceId: data.workspaceId,
+					workspaceInstanceId: data.workspaceInstanceId,
+					sessionID: data.sessionID,
+					timestamp: data.timestamp
+				},
+			};
+		case 'notification:actionExecuted':
```

Comment:
Maybe it should be part of `vscode_notification` as well

Reply:
done